docs/guide/features.md

```diff
@@ -510,11 +511,11 @@ init().then((instance) => {
 })
 ```
 
-The init function can also take the `imports` object which is passed along to `WebAssembly.instantiate` as its second argument:
+The init function can also take an imports object which is passed along to [`WebAssembly.instantiate`](https://developer.mozilla.org/en-US/docs/WebAssembly/JavaScript_interface/instantiate) as its second argument:
 
 ```js
 init({
-  imports: {
+  someModule: {
```

Comment:
Maybe it is better to align with MDN here, they are using `imports` so I would suggest we stick with the current naming: https://developer.mozilla.org/en-US/docs/WebAssembly/JavaScript_interface/instantiate#first_overload_example

Reply:
I'll note that the spec uses "js" as the sample module name: https://webassembly.github.io/spec/js-api/#sample

(MDN also tripped me up in trying to understand the parameter, so I'd love to not just do what they do. :)

That case having been made, if you want "imports", sure.

Reply:
I let @sapphi-red weight on this one. Thanks for the changes!

Reply:
SG. Thanks for the review and providing Vite!

Reply:
I'll select `imports` as I assume more people would read MDN than the spec.